src/index.ts

```diff
@@ -98,6 +98,7 @@ const program = new Command(packageJson.name)
     .option('-u, --username <username>', 'User name for the virtual machine that will be created as part of the solution')
     .option('-p, --password <password>', 'Password for the virtual machine that will be created as part of the solution')
     .option('--sshFilePath <sshFilePath>', 'Path to the ssh file path that will be used by standard deployment')
+    .option('--fxUrl <fxUrl>', 'Azure function app url for the diagnostics service')
```

Comment:
nit: recommend renaming this to diagnosticsUrl or something instead of calling it function app url